packages/flutter/lib/src/material/radio.dart

```diff
@@ -328,12 +375,75 @@ class Radio<T> extends StatelessWidget {
   /// Defaults to false.
   final bool useCupertinoCheckmarkStyle;
 
+  /// {@macro flutter.widget.RawRadio.groupRegistry}
+  ///
+  /// Unless provided, the [BuildContext] will be used to look up the ancestor
+  /// [RadioGroupRegistry].
+  final RadioGroupRegistry<T>? groupRegistry;
+
   final _RadioType _radioType;
 
+  /// {@template flutter.material.Radio.enabled}
+  /// Whether this widget is interactable.
```

Comment:
Nit: interactive? Looking it up, I guess interactable is a word too, so either is fine.